parser/src/main/kotlin/com/linkedin/dex/parser/JUnit3Extensions.kt

```diff
@@ -65,7 +65,7 @@ private fun findJUnit3Tests(dexFiles: List<DexFile>, testNames: MutableSet<TestM
 private fun DexFile.findJUnit3Tests(descriptors: MutableSet<String>): List<TestMethod> {
     val testClasses = findClassesWithSuperClass(descriptors)
     return createTestMethods(testClasses, { classDef, _ -> findMethodIds(classDef) })
-            .filter { it.testName.contains("#test") }
+            .filter { it.testName.contains("#test") and !it.testName.endsWith("#testFoo") }
```

Comment:
This seems like a strange behavior to bake into a library - I imagine your code may use testFoo as a fake test, but from the libraries perspective (as well ass junit afaik) it is a valid test, so I'm not ure we should be making assumptions about intention. Thoughts?